BitwardenShared/UI/Platform/Application/AppProcessor.swift

```diff
@@ -423,11 +424,23 @@ extension AppProcessor: Fido2UserInterfaceHelperDelegate {
     }
 
     func onNeedsUserInteraction() async throws {
-        if let fido2AppExtensionDelegate = appExtensionDelegate as? Fido2AppExtensionDelegate,
-           !fido2AppExtensionDelegate.flowWithUserInteraction {
+        guard let fido2AppExtensionDelegate = appExtensionDelegate as? Fido2AppExtensionDelegate else {
+            return
+        }
+
+        if !fido2AppExtensionDelegate.flowWithUserInteraction {
             fido2AppExtensionDelegate.setUserInteractionRequired()
             throw Fido2Error.userInteractionRequired
         }
+
+        // WORKAROUND: We need to wait until the view controller appears in order to perform any
+        // action that needs user interaction or it might not show the prompt to the user.
+        // E.g. without this there are certain devices that don't show the FaceID prompt
```

Comment:
⛏️ "E.g." means "for example" ("*exempli gratia*"); if you want "that is" or "in other words", that'd be "i.e." ("*id est*")

Reply:
I wanted exampli gratia, as there are other cases in which this happens as setting up the PIN.

Reply:
Cool, wasn't sure!